client/src/game/types/CardState.ts

```diff
@@ -9,10 +9,14 @@ export default interface CardState {
   readonly rankClueMemory: ClueMemory;
   readonly colorClueMemory: ClueMemory;
 
-  // possibleCards[suitIndex][rank] = how many cards of this suitIndex and rank could this be?
+  // matching[suitIndex][rank] = does the card of this suitIndex and rank match our clues?
+  // unseenCards[suitIndex][rank] = how many cards of this suitIndex and rank could this be?
   // NOTE: we're using an array as a map, so there will be empty spaces for ranks
   // that are not valid card ranks (e.g. 0, or 6 in Up or Down)
-  readonly possibleCards: ReadonlyArray<readonly number[]>;
+  readonly matchingCards: ReadonlyArray<readonly boolean[]>;
```

Comment:
not a fan of this name either. I see the comment, but what does "match our clues" mean? I.e. if i touch a pink with 2, any 2 and any pink "match" this clue. This map is stored on an array, so be careful with non-contiguous map/filter